src/traces/ohlc/ohlc_defaults.js

```diff
@@ -24,17 +22,13 @@ module.exports = function handleOHLC(traceIn, traceOut, coerce, layout) {
     var handleCalendarDefaults = Registry.getComponentMethod('calendars', 'handleTraceDefaults');
     handleCalendarDefaults(traceIn, traceOut, ['x'], layout);
 
-    len = Math.min(open.length, high.length, low.length, close.length);
+    if(!(open && high && low && close)) return;
+
+    var len = Math.min(open.length, high.length, low.length, close.length);
 
-    if(x) {
-        len = Math.min(len, x.length);
-        if(len < x.length) traceOut.x = x.slice(0, len);
-    }
+    if(x) len = Math.min(len, x.length);
 
-    if(len < open.length) traceOut.open = open.slice(0, len);
-    if(len < high.length) traceOut.high = high.slice(0, len);
-    if(len < low.length) traceOut.low = low.slice(0, len);
-    if(len < close.length) traceOut.close = close.slice(0, len);
+    traceOut._inputLength = len;
```

Comment:
Any reason for naming this `_inputLength`? Parcoords and splom use `_commonLength` to stash their common `dimensions[i].values` length - which I guess is a little different than the finance trace case here and the regular `traceOut._length` used in (all?) other traces.

Reply:
This will be converted to [`_length`](https://github.com/plotly/plotly.js/blob/3d65de3adb20ca0c75e9ecb67739c5d801cc3b24/src/plots/cartesian/set_convert.js#L408) when I de-transform these traces, but I didn't want to use that (or `_commonLength`) here since the post-transform traces have different lengths.